module/VuFind/src/VuFind/ChannelProvider/ChannelLoader.php

```diff
@@ -69,6 +70,35 @@ public function __construct(
     ) {
     }
 
+    /**
+     * Add configuration values needed by the templates to the view context
+     *
+     * @param array $context String-keyed map of values for the View
+     *
+     * @return array
+     */
+    protected function addConfigToContext($context)
+    {
+        for ($i = 0; $i < count($context['channels']); $i++) {
```

Comment:
Should we add a failsafe here?

```suggestion
        for ($i = 0; $i < count($context['channels'] ?? []); $i++) {
```